EasyLearn/config/database.yml

```diff
@@ -4,19 +4,19 @@
 postgresql: &postgresql
   adapter: postgresql
   host: localhost
-  username: username
-  password: password
+  username: p_final
+  password: p_finalBC2021
```

Comment:
Your database configuration file still appears on GitHub.

I recommend you clear the cache, and do a git amend, so that file no longer appears on GitHub.

```
git rm -r --cached .
git add .
git commit --amend --no-edit
git push --force-with-lease origin your-branch
```

Reply:
Oooh yeah, you're right, thanks for the steps, I already updated.